website/src/app/[lang]/[region]/(website)/partners/(sections)/ngocard.tsx

```diff
@@ -0,0 +1,370 @@
+import { WebsiteLanguage } from '@/i18n';
+import { UsersIcon } from '@heroicons/react/24/solid';
+import { Translator } from '@socialincome/shared/src/utils/i18n';
+import {
+	Badge,
+	Card,
+	CardContent,
+	CardFooter,
+	CardHeader,
+	CardTitle,
+	Dialog,
+	DialogContent,
+	DialogHeader,
+	DialogTitle,
+	DialogTrigger,
+	HoverCard,
+	HoverCardContent,
+	HoverCardTrigger,
+	Separator,
+	Typography,
+} from '@socialincome/ui';
+import { FontColor } from '@socialincome/ui/src/interfaces/color';
+import { cn } from '@socialincome/ui/src/lib/utils';
+import { SL } from 'country-flag-icons/react/1x1';
+import Image from 'next/image';
+import Link from 'next/link';
+import { ReactElement } from 'react';
+import SdgIcon from '../(assets)/sdg-circle.svg';
+
+type quoteType = {
```

Comment:
Types should always start with an uppercase letter

Reply:
Resolved...with the amount of different languages I am writing for different projects, keeping track of the conventions for that particular project is becoming hard...sorry for this, I'll try to refer to how it is done elsewhere instead of taking a guess